### PR TITLE
Change default font to Open Sans - RSC-80 

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-80

Simple change to `_nx-text-helpers.scss` to update the `font-family` mixin.

### CODE REVIEW

File: lib/src/scss-shared/_nx-text-helpers.scss [hunk 1]
```diff
@@ -7,7 +7,8 @@
 @import 'nx-variables';
 
 @mixin nx-font-family() {
-  font-family: Proxima Nova, Helvetica Neue, Helvetica, sans-serif;
+  @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap');
```
Comment (rpokorny) [inline lib/src/scss-shared/_nx-text-helpers.scss:10]: Cannot have external URL references in our products

Comment (glenhunter) [reply lib/src/scss-shared/_nx-text-helpers.scss]: > Cannot have external URL references in our products

This required a little bit more rework than I expected: https://github.com/sonatype/sonatype-react-shared-components/pull/4/commits/26df72ac46e83bcc98402b7b2e23b4257be1ee84

File: lib/src/base-styles/_nx-typography.scss [hunk 2]
```diff
@@ -7,6 +7,27 @@
...
+  font-weight: 600;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Open Sans";
+  src: url("../assets/fonts/open-sans-v17-latin-700.woff") format("woff");
+  font-weight: bold;
+  font-style: normal;
+}
```
Comment (rpokorny) [inline lib/src/base-styles/_nx-typography.scss:29]: No italic styles?

Comment (glenhunter) [reply lib/src/base-styles/_nx-typography.scss:29]: > No italic styles?

https://github.com/sonatype/sonatype-react-shared-components/pull/4/commits/13c13e0ed36b7679ec534556943584bcb28208ed

Comment (rpokorny) [reply lib/src/base-styles/_nx-typography.scss:29]: > 13c13e0

No _**bold italic**_ styles? :grin:

Comment (regoarrarr) [reply lib/src/base-styles/_nx-typography.scss:29]: > > [13c13e0](https://github.com/sonatype/sonatype-react-shared-components/commit/13c13e0ed36b7679ec534556943584bcb28208ed)
> 
> No _**bold italic**_ styles? 😁

LOL

Comment (glenhunter) [reply lib/src/base-styles/_nx-typography.scss:29]: > > [13c13e0](https://github.com/sonatype/sonatype-react-shared-components/commit/13c13e0ed36b7679ec534556943584bcb28208ed)
> 
> No _**bold italic**_ styles? 😁

Well no, we have definitely never had specific bold italic fonts installed. 
BUT if the feeling is that we should have then should we also consider adding Light, Light Italic, Semi-bold Italic? Just to be complete?

The Extra Bold is really ugly IMO, no way should we add that.

**Pull request merged**